### PR TITLE
Terraform installation

### PR DESCRIPTION
**Pull request by AIexei**
https://trello.com/c/WqD7Vcva

### CODE REVIEW

File: tank/main.py [hunk 1]
```diff
@@ -11,15 +11,19 @@
...
 
 
 def _default_config() -> Dict:
     config = init_defaults('tank',
                            'log.logging')
 
     config['tank'] = {
-        'terraform_run_command': '/usr/local/bin/terraform',
-        'terraform_inventory_run_command': '/usr/local/bin/terraform-inventory',
+        'terraform_run_command': 'terraform',
```
Comment (Eenae) [inline tank/main.py]: Will it work? Are they on $PATH?
I suggest installing into subdir `bin` of `app.user_dir` and adding this subdir to the `PATH` at `app_env`

Comment (AIexei) [reply tank/main.py]: @Eenae I unpack archive to `app.user_dir` then I add this directory to `PATH` variable.

Comment (AIexei) [reply tank/main.py]: Should I unpack it to `bin` or remain it as it is?

Comment (Eenae) [reply tank/main.py]: lets use `bin`

Comment (AIexei) [reply tank/main.py]: @Eenae done

**Pull request merged**